src/modules/previewpane/MarkdownPreviewHandler/MarkdownTelemetry.cs

```diff
@@ -20,12 +20,12 @@ public class MarkdownTelemetry : TelemetryBase
         /// <summary>
         /// ETW event name when markdown is previewed.
         /// </summary>
-        private const string MarkdownFilePreviewedEventName = "PowerPreview_PreviewPane_MDRendererPreviewed";
+        private const string MarkdownFilePreviewedEventName = "PowerPreview_MDRenderer_Previewed";

```

Comment:
thanks making the changes! Looks good to me. Do you think we can make the similar change to the settings.cpp file (NOT require since it is out of the scope of the PR)

Reply:
I am more inclined of putting it in a separate PR. This PR is focused on C# telemetry.

Reply:
sounds good!!!